test/extended/testdata/jenkins-plugin/build-job.xml

```diff
@@ -27,6 +27,22 @@
       <authToken></authToken>
       <followLog>true</followLog>
       <checkForTriggeredDeployments>true</checkForTriggeredDeployments>
+
```

Comment:
ok cool - thx - and yeah, to reiterate what i said in the preface to this latest review, unless you are close or something, definitely switch to the single `g.It` if that is the by far easiest or only way to allow us to keep Jenkins up for the duration of the tests